docs/digger/installation/known-issues.asciidoc

```diff
@@ -0,0 +1,29 @@
+== Known limitations
+
+=== HTTP proxy with mac Jenkins node
+When adding a mac machine as a node/slave in Jenkins a direct connection is
+required, meaning a HTTP proxy is not supported.
+
+=== HTTP proxy authentication in Android builds
+When building Android apps, the pod in which the apps are built cannot use a
+proxy that requires authentication. This is due to a issue within Gradle.
+
+=== Encrypted private key cannot be provided in Ansible provisioning
+The user's private key can not be encrypted when running the Ansible playbook.
+
+Note that this does not include the key pair that should be provided when
+provisioning the macOS node, the private key of the macOS key pair should be
+encrypted with a passphrase.
+
+*Workaround*
+You can use `openssl` to remove the passphrase from your key using the
+following command.
+
+----
+openssl rsa -in ~/.ssh/id_rsa -out ~/.ssh/id_rsa_new
+----
+
+=== Android SDK Manager tools don't support HTTP proxy with authentication
+When installing the Android SDK into the SDK pod during provisioning, the
+Android sdkmanager tools used doesn't support http proxy that requires
+authentication.
```

Comment:
I think it will be easier for users to find information if we can group those issues into a few different categories. For example, there are 3 of them are about http proxy, they can be group into a category called "Issues with using Http Proxy".